apis/r/tools/install-linux-dependencies.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -euo pipefail
+# Context: .github/workflows/*.yaml
+apt-get update -y && apt-get install -y libgeos-dev
```

Comment:
Are we sure we need `libgeos` for `rgeos` anywhere?

Reply:
🤷 Probably not -- this is no more & no less than a copy from `main-old`.

Reply:
`main-old` has (IIRC) a dependency on [SeuratObject](https://cran.r-project.org/package=SeuratObject), which (used to) have a dependency on rgeos; `main` does not have a dependency on SeuratObject, and SeuratObject now suggests rgeos rather than depends on it

Reply:
Perfect, so let's nix the script, or keep it but remove its call.

Shorter CI cycles are good.  More feedback, faster.